pkg/build/admission/config.go

```diff
@@ -12,7 +12,6 @@ import (
 
 // ReadPluginConfig will read a plugin configuration object from a reader stream
 func ReadPluginConfig(pluginConfig map[string]configapi.AdmissionPluginConfig, name string, config runtime.Object) error {
-
```

Comment:
May be best to not bother with this change since it just adds noise to the diff.